packages/ui/src/components/VerificationCode/index.tsx

```diff
@@ -45,6 +65,14 @@ const StyledInput = styled.input`
   &::placeholder {
     color: ${({ theme }) => theme.colors.neutral.textWeak};
   }
```

Comment:
Proto was not clear about placeholder so I asked for clarification. Here you need to add a condition in css to check if the input is disabled then the color of placeholder should be `neutral.textWeakDisabled`